### ISSUE TITLE
[Feature Request] Define custom apidoc section independt from controller name

### ISSUE THREAD

**Issue by bastian-src** [Apipie/apipie-rails#730]
Hey! I've had a look at your documentation and I can't find a command to achieve the following:

I have an existing controller which is called `JobsController`. _apidoc_ creates a corresponding section which is called `Jobs`. Now, I want to change the section name without changing the controller name. Is there any _apipie_ command to define a custom _apidoc_ section name such that it doesn't use the controller name anymore? Something like:

```
class JobsController
  api_section 'CustomJobs'
...
```

The reason for this is that I don't want to rename the whole controller (and all its occurrences), but the corresponding documentation shall be listed under a different section name.

Maybe I just overlooked it and the functionality is already given by a command.

**Comment by ofedoren** [comment]
Hey :) Thanks for using the library and sorry for the delay. Have you tried to use
```ruby
class JobsController
  resource_description do
    name 'CustomJobs'
  end
end
```
?

**Comment by bastian-src** [comment]
Hey! thanks for your reply :)
I've tested your suggestion & it changes the name, thanks.

Unfortunately, it doesn't change the order where the entry is listed at the apidoc page. I thought it would be generated alphabetic & change the order if I change the name. Do you have an idea on how to fix this?

**Comment by vinczedani** [comment]
Hello
I have some similar question / issue when using swagger ui with the swagger json generated by apipie. The jsons always contain the tag even if I specify tags in the resource_description, or give the resource a unique name, this results in multiple occurrences of the same endpoint. Can I somehow get rid of the tag that was nat given by me?
Thank you!

**Comment by mathieujobin** [comment]
did you mean to say something here @vinczedani ?

**Comment by vinczedani** [comment]
yup I wanted to, but then I realised its not in the scope of this issue, so moved it to https://github.com/Apipie/apipie-rails/issues/759
sorry for the double

**Comment by davidwessman** [comment]
I think @vinczedani is correct that it would help to remove the default resource tag to allow the fix from @ofedoren to work.
Hopefully fixed in https://github.com/Apipie/apipie-rails/pull/881

**Comment by bastian-src** [comment]
Alright, thanks for your help! @davidwessman 
I guess I'll keep the issue open until that PR is merged.
